### PR TITLE
fix bug on generate vector 

### PR DESCRIPTION
**Pull request by ShawnShawnYou**
* fix bug on generate vector

### CODE REVIEW

**Review by inabao — APPROVED** [review]
LGTM

File: tests/fixtures/fixtures.h [hunk 1]
```diff
@@ -33,9 +33,9 @@ template <typename T, typename RT = typename std::enable_if<std::is_integral_v<T
 std::vector<RT>
 GenerateVectors(uint64_t count,
                 uint32_t dim,
+                int seed = 47,
                 T min = std::numeric_limits<T>::lowest(),
-                T max = std::numeric_limits<T>::max(),
-                int seed = 47) {
+                T max = std::numeric_limits<T>::max()) {
```
Comment (ShawnShawnYou) [inline tests/fixtures/fixtures.h:38]: p.s. The usage of this func in other function is like: GenerateVectors(count, dim, seed). So the old version assigns seed to min, triggering a bug.

Comment (jiaweizone) [reply tests/fixtures/fixtures.h:38]: the override `GenerateVectors(uint64_t count, uint32_t dim, bool need_normalize = true, int seed = 47)` also need adjust seed parms ?

**Review by LHT129 — APPROVED** [review]
LGTM

**Review by wxyucs — APPROVED** [review]
lgtm

**Comment by wxyucs** [comment]
@ShawnShawnYou please create a related issue

File: tests/fixtures/fixtures.h (hunk 1 above)
Comment (ShawnShawnYou) [reply tests/fixtures/fixtures.h:38]: done

**Pull request merged**